sdcm/stress_thread.py

```diff
@@ -191,6 +194,10 @@ def _run_cs_stress(self, loader, loader_idx, cpu_idx, keyspace_idx):  # pylint:
                 LOGGER.info('Profile content:\n%s', profile_file.read())
             cmd_runner.send_files(self.profile, os.path.join('/tmp', os.path.basename(self.profile)), delete_dst=True)
 
+        if connection_bundle_file := self.node_list[0].parent_cluster.connection_bundle_file:
+            cmd_runner.send_files(str(connection_bundle_file),
+                                  str(Path('/tmp') / connection_bundle_file.name), delete_dst=True, verbose=True)
```

Comment:
In another commit it is defined as "target_connection_bundle_file" in the "sdcm/scylla_bench_thread.py" file.
So, maybe better to define it in common place and reuse?

Reply:
I've waited for the docker based c-s to be merge, and now they share the same base class.
and it can be refactored

Reply:
Ok, are you planning to do it here?

Reply:
yes (now that you remaindered me of that)

Reply:
Still planned to be done here?

Reply:
yes, fixed